### ISSUE TITLE
AccessibilityBridge test on iOS

### ISSUE THREAD

**Issue by cyanglaz** [flutter/flutter#42669]
We should have tests to test if the accessibility bridge builds the iOS accessibility tree correct based on the semantics tree provided by the framework. It would also require a dedicated iOS device that has voice over on to run the test. 

https://github.com/flutter/flutter/issues/42270

**Comment by godofredoc** [comment]
@cyanglaz would you mind sharing the testbed requirements?

- Phone model / ios-version
- Host macos version + xcode version
- Phone configurations
- Anything else to run the test successfully

**Comment by cyanglaz** [comment]
- Phone model / ios-version
   ios 8 and above, model doesn't matter
- Host macos version + xcode version
   xcode 13 is preferred. 
- Phone configurations
   turn on the voice over.
- Anything else to run the test successfully
   No. The test im working on will be one of the XCUITest under the scenario app in engine.

**Comment by cyanglaz** [comment]
Un-assign myself from this issue as I don't have cycle to actively working on it.

**Comment by godofredoc** [comment]
Closing because the bug has not been updated for more than one year.

Feel free to reopen any bugs that you believe are important but please provide updates on how to resource the fix of the bug and what can be done to prevent the issue from not getting updates for another year.

**Comment by cyanglaz** [comment]
I don't think this has every been worked on. Reopen.
